MediaBrowser.Controller/Lyrics/LyricResponse.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+
+namespace MediaBrowser.Controller.Lyrics;
+
+/// <summary>
+/// LyricResponse model.
+/// </summary>
+public class LyricResponse
+{
+    /// <summary>
+    /// Gets or sets Metadata.
+    /// </summary>
+    public LyricMetadata Metadata { get; set; } = new();
+
+    /// <summary>
+    /// Gets or sets Lyrics.
```

Comment:
```suggestion
    /// Collection of individual lyric lines
```

Reply:
I can't make this change without the analyzer complaining.

However `Gets or sets a collection of individual lyric lines.` would work.